src/resources/io/fsq/ssp/codegen/scala/constants.ssp

```diff
@@ -8,7 +8,12 @@
 
 object ${baseName}Constants {
   #for (const <- constants)
-  <% val value = const.valueOption.map{v => "val %s: %s = %s".format(const.name, const.renderType.text, v)} %>
+  <% val value = const.valueOption.map{v => {
```

Comment:
I'm thinking we should move this higher up, either in `ScalaConst.valueOption` or better `PrimitiveRenderType.renderValue`. Is there ever a time where we _don't_ want the `L`?